test/question.test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class QuestionTest < MiniTest::Test
+  def setup
+    @question = Chiketto::Question.new 'question' => {
+        'text' => 'test question',
+        'html' => '<p>test question</p>'
+      },
+      'id' => '123456',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.